artifacthub-repo.yml

```diff
@@ -0,0 +1,27 @@
+# Artifact Hub repository metadata file
+#
+# Some settings like the verified publisher flag or the ignored packages won't
+# be applied until the next time the repository is processed. Please keep in
+# mind that the repository won't be processed if it has not changed since the
+# last time it was processed. Depending on the repository kind, this is checked
+# in a different way. For Helm http based repositories, we consider it has
+# changed if the `index.yaml` file changes. For git based repositories, it does
+# when the hash of the last commit in the branch you set up changes. This does
+# NOT apply to ownership claim operations, which are processed immediately.
+#
+repositoryID: The ID of the Artifact Hub repository where the packages will be published to (optional, but it enables verified publisher)
+owners: # (optional, used to claim repository ownership)
+  - name: Wenkai Yin
+    email: yinw@vmware.com
+  - name: Yan Wang
+    email: wangyan@vmware.com
+  - name: Miner Yang
+    email: yminer@vmware.com
+  - name: OrlinVasilev
+    email: ovasilev@vmware.com
+  - name: Shengwen Yu
+    email: yshengwen@vmware.com
+ignore: # (optional, packages that should not be indexed by Artifact Hub)
```

Comment:
If we don't need this part, how about remove or comment it?

Reply:
done